clang-tools-extra/clang-tidy/readability/ImplicitBoolConversionCheck.h

```diff
@@ -36,6 +36,7 @@ class ImplicitBoolConversionCheck : public ClangTidyCheck {
 
   const bool AllowIntegerConditions;
   const bool AllowPointerConditions;
+  const bool UseUpperCaseSuffix;
```

Comment:
`UseUpperCaseLiteralSuffix` ?

Reply:
Will this also apply to the option passed into clang-tidy

Reply:
Yes, the comment is in regard to the option name. My rationale is, that the current option name is about an upper-case suffix, but suffix of what? Adding the literal will make things clearer.